misk/src/main/kotlin/misk/web/actions/LivenessCheckAction.kt

```diff
@@ -0,0 +1,32 @@
+package misk.web.actions
+
+import com.google.common.util.concurrent.Service
+import com.google.common.util.concurrent.Service.State
+import misk.web.Get
+import misk.web.PlaintextResponseBody
+import misk.web.Response
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class LivenessCheckAction : WebAction {
+    @Inject private lateinit var services: MutableSet<Service>
+
+    @Get("/_liveness")
+    @PlaintextResponseBody
+    fun livenessCheck(): Response<String> {
+        var isAlive = true
```

Comment:
ditto here, can eliminate the mutable var by doing

``
val isAlive = services.any { it.state() == Status.FAILED || it.state() == State.TERMINATED }
``